setup.py

```diff
@@ -86,6 +86,7 @@
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
```

Comment:
Note: the CI failures for py313 are the same as for py312 and are related to the recent change in the CERN Open Data portal's handling of files attached to records via file indexes.

Test results for Python 3.13:

```console
$ tox -e py313
FAILED tests/test_cli_download_files.py::test_dry_run_from_doi - AssertionError: assert 'CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0001_file_index.json' in '\n'
FAILED tests/test_cli_get_file_locations.py::test_get_file_locations_from_recid_expand - AssertionError: assert '2A227E10-C949-E311-B033-003048FEAF50.root' in ''
FAILED tests/test_cli_get_file_locations.py::test_get_file_locations_from_recid_expand_verbose - AssertionError: assert '2A227E10-C949-E311-B033-003048FEAF50.root' in ''
FAILED tests/test_cli_get_file_locations.py::test_get_file_locations_from_doi - AssertionError: assert 'CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0001_file_index.json' in ''
FAILED tests/test_metadater.py::test_get_metadata_from_filter_metadata_two - assert 1 == 0
=============================================================================================== 5 failed, 65 passed, 10 skipped in 7.96s ===============================================================================================
py313: exit 1 (8.22 seconds) cernopendata-client> pytest pid=685218
  py313: FAIL code 1 (14.79=setup[6.57]+cmd[8.22] seconds)
  evaluation failed :( (14.84 seconds)
```

Test results for Python 3.13:

```console
$ tox -e py313
FAILED tests/test_cli_download_files.py::test_dry_run_from_doi - AssertionError: assert 'CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0001_file_index.json' in '\n'
FAILED tests/test_cli_get_file_locations.py::test_get_file_locations_from_recid_expand - AssertionError: assert '2A227E10-C949-E311-B033-003048FEAF50.root' in ''
FAILED tests/test_cli_get_file_locations.py::test_get_file_locations_from_recid_expand_verbose - AssertionError: assert '2A227E10-C949-E311-B033-003048FEAF50.root' in ''
FAILED tests/test_cli_get_file_locations.py::test_get_file_locations_from_doi - AssertionError: assert 'CMS_Run2010B_BTau_AOD_Apr21ReReco-v1_0001_file_index.json' in ''
FAILED tests/test_metadater.py::test_get_metadata_from_filter_metadata_two - assert 1 == 0
=============================================================================================== 5 failed, 65 passed, 10 skipped in 7.87s ===============================================================================================
py312: exit 1 (8.15 seconds) cernopendata-client> pytest pid=686540
  py312: FAIL code 1 (12.79=setup[4.64]+cmd[8.15] seconds)
  evaluation failed :( (12.84 seconds)
```